Gaijinizer patch/Map046.txt

```diff
@@ -140,8 +145,8 @@ to appear ...#MTLed
 > CONTEXT: Map046/events/23/pages/0/6/Dialogue/yuuma
 > CONTEXT: Map046/events/29/pages/0/6/Dialogue/yuuma
 > CONTEXT: Map046/events/30/pages/0/6/Dialogue/yuuma
-Thanks to it, the whole city has become a large
-Panic. It took a long time for evacuation .....#MTLed
+Because of that, the whole city fell into mass
+panic. The evacuation was slow...#Ron
```

Comment:
_"Thanks to that, the whole city became a big panic.
It took me a long time to evacuate..."_ 
GT and a few other translations mention that Yuuma himself is doing the evacuation. May be a false positive, but might be good to add in here.

Reply:
Based on in-game text, Yuuma was assigned to a unit of soldiers to help with the evacuation, most of whom survived. So I changed it to 

_"It took us a long time to evacuate..."_